### PR TITLE
[chore] Update StartOS Brand

### PR DESCRIPTION
**Pull request by kn0wmad**
### Describe the changes you have made in this PR

Updated branding/language for StartOS

### Type of change

- `fix`: branding

### How has this been tested?

Tested via Firefox debugging tool

### Checklist

- [x] Self-review of changed code
- [x] Manual testing
- [ ] Added automated tests where applicable (N/A)
- [ ] Update Docs & Guides (N/A
- For UI-related changes
- [x] Darkmode
- [ ] Responsive layout N/A

### CODE REVIEW

File: static/assets/icons/startos.png
Comment (reneaaron) [inline static/assets/icons/startos.png:1]: I'd heavily recommend adding some margin to the inner part of this logo to maintain some whitespace around it.

**Review by reneaaron — CHANGES_REQUESTED** [review]
Thanks for the PR, looks good! :muscle: 

Let's update the logo and :shipit:

File: static/assets/icons/startos.png
Comment (kn0wmad) [reply static/assets/icons/startos.png:1]: Fixed, ready for re-review

**Review by reneaaron — APPROVED** [review]
tACK

**Pull request merged**